README.md

```diff
@@ -98,7 +98,7 @@ will be on a best-effort basis.
 > system.
 
 Make a snapshot of the generated [`build_runner`][build_runner] build script and
-run from the snapshot instead of source to avoid problems with deleted files.
+run from the snapshot instead of the source to avoid problems with deleted files.
```

Comment:
I think that the existing `instead if source` is fine here.

Reply:
How about "from source" to tie it back to the verb?